src/Generation/GapicClientV2Generator.php

```diff
@@ -661,6 +677,19 @@ private function construct(): PhpClassMember
             ));
     }
 
+    private function transportDocText(int $transportType): string
+    {
+        if (Transport::isRestOnly($transportType)) {
+            return 'At the moment, supports only `rest`.';
+        }
+
+        if (Transport::isGrpcOnly($transportType)) {
+            return 'At the moment, supports only `grpc`.';
+        }
+
+        return 'May be either the string `rest` or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.';;
```

Comment:
remove extra semicolon 
```suggestion
        return 'May be either the string `rest` or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.';
```